onnx/shape_inference/implementation.cc

```diff
@@ -649,6 +649,10 @@ class ShapeInferenceImplBase {
     }
   }
 
+  const std::vector<std::string>& Errors() const {
```

Comment:
Not sure the naming convention for getting reference here -- shall we use getErrors() instead?

Reply:
done